### PR TITLE
RethinkDB Perf Report

### PR DESCRIPTION
**Pull request by dalanmiller**
Potentially moving performance report to a page under docs. 

rethinkdb/www#199

cc @mglukhovsky @danielmewes

### CODE REVIEW

**Comment by dalanmiller** [comment]
capture d ecran 2016-03-21 14 06 51

This needs review, but is ready to go as long as the sidebar is ready.

**Comment by mglukhovsky** [comment]
Rather than put this in a new section, can we just list "Performance report" under "RethinkDB architecture"?

Also, @dalanmiller -- we use sentence case. Would you mind updating the title of the report?

**Comment by dalanmiller** [comment]
Since we plan on doing this in the future, do you want to list every single report under "RethinkDB architecture"? I was figuring we'd order most recent first in this new section. 

Or, if it lives under architecture, _only_ the most recent version is kept there, but one could find the older ones in the rethinkdb/performance-reports repo

**Comment by mglukhovsky** [comment]
No, we should link to the most current report on the sidebar, and in each performance report include a TOC that has links to past performance reports. We should probably add a TOC to this article, and below the TOC include another section labeled "PREVIOUS PERFORMANCE REPORTS"

**Comment by dalanmiller** [comment]
Awesome, right-:o:.

**Comment by dalanmiller** [comment]
Alright, now ready again for review.

capture d ecran 2016-03-21 15 03 56

**Comment by mglukhovsky** [comment]
@dalanmiller -- I haven't reviewed the copy (yet). Is this the best place to do it?

**Comment by dalanmiller** [comment]
[Doing it here would be best](https://docs.google.com/document/d/15vPsdB8YyynQYcmvWlb_MTjBb1lDY_Z6eeIl7T_NZx0/edit#)

**Comment by mglukhovsky** [comment]
OK. I need to do an editing pass, will try to take care of it tonight (or
tomorrow morning.)

> On Mar 21 2016, at 3:32 pm, Daniel Alan Miller
&lt;notifications@github.com&gt; wrote:  

>

> [Doing it here would be best](https://docs.google.com/document/d/15vPsdB8Yyy
nQYcmvWlb_MTjBb1lDY_Z6eeIl7T_NZx0/edit#)

>

> —  
You are receiving this because you were mentioned.  
Reply to this email directly or [view it on GitHub](https://github.com/rethink
db/docs/pull/1047#issuecomment-199518529)

File: _jekyll/_data/docs-nav.yml [hunk 1]
```diff
@@ -196,3 +196,6 @@
       - name: Frequently Asked Questions
         permalink: faq
         url: /docs/faq/
+      - name: Performance Report
```
Comment (mglukhovsky) [inline _jekyll/_data/docs-nav.yml]: Sentence case capitalization:

> Frequently asked questions

**Pull request merged**